Merge and energy consumption pages pre-merge updates (#6599)
ethereum/ethereum-org-website: src/content/upgrades/merge/index.md

template: upgrade
sidebar: true
image: ../../../assets/upgrades/merge.png
summaryPoint1: Soon the current Ethereum Mainnet will merge with the Beacon Chain proof-of-stake system.
summaryPoint2: This will mark the end of proof-of-work for Ethereum, and the full transition to proof-of-stake.
summaryPoint3: This sets the stage for future scaling upgrades including data sharding.

> summaryPoint4: With The Merge, Ethereum's energy consumption drops by 99.95%. Welcome to a new greener Ethereum.

Reviewer: I like the idea here but not sure if we should be including vague statements like this in the summary points

Author: Could you clarify what you mean here? These each include specifics, and together summarize the merge.

> summaryPoint4: With The Merge, Ethereum's energy consumption drops by 99.95%. Welcome to a new greener Ethereum.

Reviewer 2: ```suggestion
summaryPoint4: The Merge will reduce Ethereum's energy consumption by ~99.95%. Welcome to a new greener Ethereum.
```

> summaryPoint4: The Merge will reduce Ethereum's energy consumption by ~99.95%. Welcome to a new greener Ethereum.

Author: Eh, not sure I love that look... but still, I think it helps keep things friendly and really drill home the energy impact.

> summaryPoint4: The Merge will reduce Ethereum's energy consumption by ~99.95%. Welcome to a new greener Ethereum.

Reviewer: As I said, not a hill I'd die on 😜 I prefer two sentences if you'd like to keep it in.

---

<UpgradeStatus dateKey="page-upgrades-merge-date">

> The Merge is the most significant and high-stakes upgrade in the history of Ethereum. Extensive testing and bug bounties were undertaken to ensure a safe transition to proof-of-stake.

Reviewer 2: I like this update but I also think the previous explanation was pretty solid:

> This eliminates the need for energy-intensive mining, and instead secures the network using staked ether. A truly exciting step in realizing the Ethereum vision – more scalability, security, and sustainability.

I wonder if there's an aspect of that we could keep? Perhaps down in the explanation below?

> The Merge is one of the most significant and high-stakes upgrades in the history of Ethereum. Extensive testing and bug bounties have been underway for years to ensure a safe transition to proof-of-stake.

Reviewer: The most?

```suggestion
The Merge is the most significant and high-stakes upgrade in the history of Ethereum. Extensive testing and bug bounties were undertaken to ensure a safe transition to proof-of-stake.
```

> This process is in its final stages, with plans to undergo The Merge on a few public testnets before finally moving forward with Mainnet. If you’re as excited about The Merge as we are, follow the [EF Blog](https://blog.ethereum.org) or the client communication channels for the latest official word on _When Merge?_

Reviewer: > If you’re as excited about The Merge as we are, follow the [EF Blog](https://blog.ethereum.org) or the client communication channels for the latest official word on _When Merge?_

This tone feels a little off compared to the rest of the doc

Reviewer 2: Agreed, e.g. we don't typically refer to "we" as in:

> If you’re as excited about The Merge as we are

Although I don't dislike the attempt to make things friendly.

Author: Hah that was the goal =)
```suggestion
This process is in its final stages, with plans to undergo The Merge on a few public testnets before finally moving forward with Mainnet. If you're excited about The Merge, follow the [EF Blog](https://blog.ethereum.org) or the client communication channels for the latest official word on _When Merge?_
```

</UpgradeStatus>

> ## What is The Merge? {#what-is-the-docking}

Reviewer 2: ```suggestion
## What is The Merge? {#what-is-the-merge}
```
Should we update these for all languages? Probably...

Author: Yes please... been updating these for the most part, but I know we were hesitant back when "docking" was deprecated to avoid breaking hash links. I think it's worth just updating these to reduce confusion.

> The Merge represents the merger of the existing execution layer of Ethereum (the Mainnet we use today) with its new proof-of-stake consensus layer, the Beacon Chain. This eliminates the need for energy-intensive mining, and instead secures the network using staked ether. A truly exciting step in realizing the Ethereum vision – more scalability, security, and sustainability.

Reviewer: > The Merge represents the merger

Awkward phrasing

> This eliminates the need

this unclear

```suggestion
The Merge represents the joining of the existing execution layer of Ethereum (the Mainnet we use today) with its new proof-of-stake consensus layer, the Beacon Chain. It eliminates the need for energy-intensive mining and instead secures the network using staked ETH. A truly exciting step in realizing the Ethereum vision – more scalability, security, and sustainability.
```

> The Merge represents the merger of the existing execution layer of Ethereum (the Mainnet we use today) with its new proof-of-stake consensus layer, the Beacon Chain.

Reviewer 2: ```suggestion
The Merge represents the merger of the existing execution layer of Ethereum (the Mainnet we use today) with its new proof-of-stake consensus layer, the Beacon Chain. This eliminates the need for energy-intensive mining, and instead secures the network using staked ether. A truly exciting step in realizing the Ethereum vision – more scalability, security, and sustainability.
```
Related to [this comment](https://github.com/ethereum/ethereum-org-website/pull/6599/files#r896747572) - maybe we add this back here?

Author: Like puzzle pieces, I like it =)

> It's important to remember that initially, the [Beacon Chain](/upgrades/beacon-chain/) (the new "consensus layer") shipped separately from [Mainnet](/glossary/#mainnet). Ethereum Mainnet (the "execution layer") - with all it's accounts, balances, smart contracts, and blockchain state - continues to be secured by [proof-of-work](/developers/docs/consensus-mechanisms/pow/), even while the Beacon Chain runs in parallel using [proof-of-stake](/developers/docs/consensus-mechanisms/pos/). The approaching Merge is when these two systems finally come together, and proof-of-work is replaced permanently by proof-of-stake.

Reviewer: > It's important to remember that initially, the [Beacon Chain](/upgrades/beacon-chain/) (the new "consensus layer")

We already tell people that the Beacon Chain === consensus layer in the paragraph above and that Mainnet === execution layer.

```suggestion
It's important to remember that initially, the [Beacon Chain](/upgrades/beacon-chain/) shipped separately from [Mainnet](/glossary/#mainnet). Ethereum Mainnet with all it's accounts, balances, smart contracts, and blockchain state - continues to be secured by [proof-of-work](/developers/docs/consensus-mechanisms/pow/), even while the Beacon Chain runs in parallel using [proof-of-stake](/developers/docs/consensus-mechanisms/pos/). The approaching Merge is when these two systems finally come together, and proof-of-work is replaced permanently by proof-of-stake.
```

Author: ```suggestion
It's important to remember that initially, the [Beacon Chain](/upgrades/beacon-chain/) shipped separately from [Mainnet](/glossary/#mainnet). Ethereum Mainnet - with all it's accounts, balances, smart contracts, and blockchain state - continues to be secured by [proof-of-work](/developers/docs/consensus-mechanisms/pow/), even while the Beacon Chain runs in parallel using [proof-of-stake](/developers/docs/consensus-mechanisms/pos/). The approaching Merge is when these two systems finally come together, and proof-of-work is replaced permanently by proof-of-stake.
```

> It's important to remember that initially, the [Beacon Chain](/upgrades/beacon-chain/) (the new "consensus layer") shipped separately from [Mainnet](/glossary/#mainnet). Ethereum Mainnet (the "execution layer") continues to be secured by [proof-of-work](/developers/docs/consensus-mechanisms/pow/), even while the Beacon Chain runs in parallel using [proof-of-stake](/developers/docs/consensus-mechanisms/pos/). The approaching Merge is when these two systems finally come together, and proof-of-work is replaced permanently by proof-of-stake.

Reviewer 2: I think we could do a better job explaining what the execution layer is / what aspects of Mainnet will remain.

Maybe instead of:
> Ethereum Mainnet (the "execution layer") continues to be secured by proof-of-work

Something along the lines of :
> Ethereum Mainnet (the "execution layer") - with all it's accounts, balances, smart contracts, and blockchain state -  continues to be secured by proof-of-work

Something that gets across the point that the execution layer is where essentially all user-facing blockchain state lives.

Author: ```suggestion
It's important to remember that initially, the [Beacon Chain](/upgrades/beacon-chain/) (the new "consensus layer") shipped separately from [Mainnet](/glossary/#mainnet). Ethereum Mainnet (the "execution layer") - with all it's accounts, balances, smart contracts, and blockchain state - continues to be secured by [proof-of-work](/developers/docs/consensus-mechanisms/pow/), even while the Beacon Chain runs in parallel using [proof-of-stake](/developers/docs/consensus-mechanisms/pos/). The approaching Merge is when these two systems finally come together, and proof-of-work is replaced permanently by proof-of-stake.
```

> Imagine Ethereum is a space ship that isn’t quite ready for an interstellar voyage. With the Beacon Chain the community has built a new engine and a hardened hull. After significant testing, it's almost time to hot-swap in the new engine for the old mid-flight. This will merge the new more efficient engine into the existing ship, ready to put in some serious lightyears and take on the universe.

Reviewer: nerd

```suggestion
Let's consider an example. Imagine Ethereum is a spaceship that isn't quite ready for an interstellar voyage. With the Beacon Chain, the community has built a new engine and a hardened hull. After significant testing, it's almost time to hot-swap the new engine for the old mid-flight. This will merge the new, more efficient engine into the existing ship, ready to put in some serious lightyears and take on the universe.
```

Author: Wasn't my original copy

> Let's consider an example. Imagine Ethereum is a spaceship that isn't quite ready for an interstellar voyage. With the Beacon Chain, the community has built a new engine and a hardened hull. After significant testing, it's almost time to hot-swap the new engine for the old mid-flight. This will merge the new, more efficient engine into the existing ship, ready to put in some serious lightyears and take on the universe.

Reviewer 2: I like the example 😄 🚀

> Let's consider an example. Imagine Ethereum is a spaceship that isn't quite ready for an interstellar voyage. With the Beacon Chain, the community has built a new engine and a hardened hull. After significant testing, it's almost time to hot-swap the new engine for the old mid-flight. This will merge the new, more efficient engine into the existing ship, ready to put in some serious lightyears and take on the universe.

Reviewer 2: ```suggestion
Let's consider an analogy. Imagine Ethereum is a spaceship that isn't quite ready for an interstellar voyage. With the Beacon Chain, the community has built a new engine and a hardened hull. After significant testing, it's almost time to hot-swap the new engine for the old mid-flight. This will merge the new, more efficient engine into the existing ship, ready to put in some serious lightyears and take on the universe.
```
Nitpicking here but I think "analogy" is more fitting vs. "example".

## Merging with Mainnet {#merging-with-mainnet}

> Since [genesis](/history/#frontier), Mainnet has been secured by proof-of-work. This is the Ethereum blockchain we're all used to—it contains every transaction, smart contract, and balance since it began in July 2015.

Author: ```suggestion
Since [genesis](/history/#frontier), proof-of-work has secured Mainnet. This is the Ethereum blockchain we're all used to—it contains every transaction, smart contract, and balance since it began in July 2015.
```

> Since genesis, Mainnet has been secured by proof-of-work. This is the main Ethereum blockchain we're all used to—it contains every transaction, smart contract, and balance since it began in July 2015.

Reviewer 2: ```suggestion
Since [genesis](/history/#frontier), Mainnet has been secured by proof-of-work. This is the main Ethereum blockchain we're all used to—it contains every transaction, smart contract, and balance since it began in July 2015.
```

Reviewer 2: Perhaps not a super necessary link given you do mention "July 2015".

Author: Ooh no, I like it.

Author: ```suggestion
Since [genesis](/history/#frontier), Mainnet has been secured by proof-of-work. This is the Ethereum blockchain we're all used to—it contains every transaction, smart contract, and balance since it began in July 2015.
```

> Since genesis, Mainnet has been secured by proof-of-work. This is the main Ethereum blockchain we're all used to—it contains every transaction, smart contract, and balance since it began in July 2015.

Reviewer 2: ```suggestion
Since genesis, Mainnet has been secured by proof-of-work. This is the Ethereum blockchain we're all used to—it contains every transaction, smart contract, and balance since it began in July 2015.
```

> Since genesis, Mainnet has been secured by proof-of-work. This is the main Ethereum blockchain that we're all used to—it contains every transaction, every smart contract, and every balance since it began in July 2015.

Reviewer: ```suggestion
Since genesis, Mainnet has been secured by proof-of-work. This is the main Ethereum blockchain we're all used to—it contains every transaction, smart contract, and balance since it began in July 2015.
```

> During this growing history, developers have been hard at work preparing for an eventual transition away from proof-of-work to proof-of-stake instead. On December 1, 2020, the Beacon Chain was created, which has since existed as a separate blockchain to Mainnet, running in parallel.

Reviewer 2: ```suggestion
During this growing history, developers have been hard at work preparing for an eventual transition away from proof-of-work to proof-of-stake. On December 1, 2020, the Beacon Chain was created, which has since existed as a separate blockchain to Mainnet, running in parallel.
```

> During this growing history, developers have been hard at work preparing for an eventual transition away from proof-of-work to proof-of-stake instead. On December 1, 2020, the Beacon Chain was created, which has since existed as a separate blockchain to Mainnet, running in parallel.

Reviewer 2: > During this growing history

For some reason this feels a bit strange, perhaps we could reword this, e.g. "throughout Ethereum's history"?

> During this growing history, developers have been hard at work preparing for an eventual transition away from proof-of-work, to proof-of-stake instead. On December 1, 2020 the Beacon Chain was born, which has since lived as a separate blockchain to Mainnet, running in parallel.

Reviewer: We should avoid personification here as it isn't as widely understood cross-culture. (Though, not sure if 'created' is technically correct either 🤔)

```suggestion
During this growing history, developers have been hard at work preparing for an eventual transition away from proof-of-work to proof-of-stake instead. On December 1, 2020, the Beacon Chain was created, which has since existed as a separate blockchain to Mainnet, running in parallel.
```

> The Beacon Chain has not been processing Mainnet transactions, and for all intents and purposed has simply been coming to consensus on the state of itself... which is great and all, but after extensive testing, the time for it to come to consensus on more than just itself is rapidly approaching.

Reviewer: simplify/tone

```suggestion
The Beacon Chain has not been processing Mainnet transactions and has only reached consensus on its own state. Still, after extensive testing, the time for it to come to consensus on more than just itself is rapidly approaching.
```

Reviewer 2: Agreed. I like this change, though:

> the time for it to come to consensus on more than just itself is rapidly approaching.

I think this could still be reworded to explain what "more than just itself" is. e.g. coming to consensus on additional aspects, like transactions & blocks.

Author: Feel like the simplification trims part of the message here... how about something like this?
```suggestion
The Beacon Chain has not been coming to agreement regarding Mainnet transactions, but instead has been reaching consensus on the state of itself. This includes agreeing on who the active validators are and also their account balances.

After extensive testing, the time for the Beacon Chain to come to consensus on more than just the balance of validators is rapidly approaching. Soon this will be the consensus engine for all network data, including execution layer transactions and all account balances.
```

Reviewer: I see your point @wackerow. What about this?

```suggestion
The Beacon Chain has not been processing Mainnet transactions. Instead, it has been reaching consensus on its own state by agreeing on active validators and their account balances. After extensive testing, the Beacon Chain's time to reach consensus on more is rapidly approaching. After The Merge, the Beacon Chain will be the consensus engine for all network data, including execution layer transactions and account balances.
```

> The Merge itself represents the official switch to using the Beacon Chain as the official means of block production. Mining will no longer be the means of producing valid blocks, and instead this role will land with the proof-of-stake validators who will then be responsible for processing the validity of all transactions, and proposing blocks.

Reviewer: Same as above, the land metaphor doesn't translate

```suggestion
The Merge represents the official switch to using the Beacon Chain as the official means of block production. Mining will no longer be the means of producing valid blocks. Instead, the proof-of-stake validators assume this role and will be responsible for processing the validity of all transactions and proposing blocks.
```

Reviewer 2: Agreed. I also don't love:

> the official means of block production

But not sure how that could be best reworded.

Author: Sure, I do like the use of "engine" in this setting

Author: Though a thought on 'metaphors not translating' in general though... We use metaphors in other places. My understanding is that translators have been doing more than just "translating" but also "interpreting" the content so their translation makes sense. Curious thoughts from others, but do we need to be that restrictive?

Author: ```suggestion
The Merge represents the official switch to using the Beacon Chain as the engine of block production. Mining will no longer be the means of producing valid blocks. Instead, the proof-of-stake validators assume this role and will be responsible for processing the validity of all transactions and proposing blocks.
```

> No history is lost. As Mainnet is merged with the Beacon Chain, the entire transactional history of Ethereum comes with it. You don't need to do anything. Your funds are safe.

Reviewer: ```suggestion
No history is lost. As Mainnet gets merged with the Beacon Chain, it will also merge the entire transactional history of Ethereum. You don't need to do anything. Your funds are safe.
```

<InfoBanner>
This transition to proof-of-stake will come with some changes to the way ether is supplied. Learn more about <a href="/upgrades/merge/eth-issuance/">ether issuance before and after The Merge</a>.
</InfoBanner>

## What to I need to do to get ready? {#preparing-for-the-merge}

The Merge is one of the most significant and anticipated upgrades in the history of Ethereum, and although in the long-term its impact will be felt by everyone, in the near-term some folks will need to take action to be fully prepared.

### Users and holders {#users-holders}

**You do not need to anything to protect your funds entering The Merge.**

> _This bears repeating_: As a user or holder of ETH or any other digital asset on Ethereum, as well as non-node-operating stakers, **you do not need to do anything with your funds or wallet before The Merge.**

Author: Yes, because I want _pooled_ stakers to know they fall into that category. I made this explicit below as well by stating "Staking node operators and providers" but would prefer to keep it in here too in case users don't keep reading

> _This bears repeating_: As a user or holder of ETH or any other digital asset on Ethereum, as well as non-node-operating stakers, **you do not need to do anything with your funds or wallet before The Merge.**

Reviewer: No worries. Could you replace bears repeating? It's not something non-Native will understand easily and it won't translate.

```suggestion
_This is worth repeating_: As a user or holder of ETH (or any other digital asset on Ethereum), as well as non-node-operating stakers, **you do not need to do anything with your funds or wallet before The Merge.**
```

> For everyday users, holders, or non-node-operating stakers there is **nothing you need to do**, but a few things you should be aware of heading into The Merge. [More on this below.](#users-holders)

Reviewer 2: I feel like this this should somehow be bold or a louder callout, as this is probably the bulk of readers.

Despite swapping out proof-of-work, the entire history of Ethereum since genesis remains intact and unaltered after the transition to proof-of-stake. Any funds being protected by your own wallet before The Merge will still be accessible in the same manner after The Merge. No action is required to "upgrade" on your part.

As we approach The Merge of Ethereum Mainnet, **you should be on high alert for scams trying to take advantage of users during this transition.** Do not send your ETH anywhere in attempt to "upgrade to ETH2." There is no "ETH2" token, and there is nothing more you need to do for your funds to remain safe.

[More on Ethereum security](/security/#eth2-token-scam)

### Node operators and dapp developers {#node-operators-dapp-developers}

<ExpandableCard
title="Staking node operators and providers"
contentPreview="If you are a staker running your own node setup, or a node infrastructure provider, there are a few things you need to be aware of to be prepared for The Merge."
id="staking-node-operators">
Key action items include:

> - Running _both_ a consensus layer client as well as an execution layer client. Third-party endpoints to obtain execution data will be unavailable after The Merge.

Reviewer: ```suggestion
- Running _both_ a consensus layer client and an execution layer client; third-party endpoints to obtain execution data will be unavailable after The Merge.
```

> - Running _both_ a consensus layer client as well as an execution layer client. Third-party endpoints to obtain execution data will be unavailable after The Merge.

Reviewer: Weird spacing here

<img width="719" alt="Screenshot 2022-06-15 at 22 17 52" src="https://user-images.githubusercontent.com/62268199/173931594-71aa4e3b-39c3-4199-82b4-a5bad6ef86e5.png">

Author: Agree. This seems to be a bug when using components in mdx, where the first lines (without an empty line as a spacer) after the opening component tag is treated one way, and the rest are parsed as markdown. Will patch this by adjusting the syntax, but we should open a separate issue about this to fix that issue more broadly.

> - Authenticating both execution layer and consensus layer clients with a shared JWT secret so they can securely communicate with one another.

Reviewer: ```suggestion
- Authenticate execution and consensus layer clients with a shared JWT secret so they can securely communicate with one another.
```

> Not completing the above items will result in your node being seen as "offline" after The Merge until both layers are synced and authenticated.

Reviewer: This should be nested under the 2nd list item or else this forms two different  lists in HTML (a11y)

Author: Pulling this line below the list (since it applies to both the first two, don't feel it should be nested under one of them). Changing these to a numbered list, and in the "Not completing the above..." just updating to "Not completing the first two items above..."

> - Setting a `fee recipient` address to direct your earned transaction fee tips/MEV to.

Reviewer: action verbs are better for making people act on instructions

```suggestion
- Set a `fee recipient` address to direct your earned transaction fee tips/MEV.
```

Reviewer 3: "Set a `fee recipient` address to **receive** your earned transaction fee tips/MEV" ?

> You probably are already running an EL client, such as Geth, Erigon, Besu or Nethermind. Up until The Merge, an EL client was enough to receive, properly validate, and propagate blocks being gossiped by the network. _After The Merge_, the validity of transactions contained within an execution payload will also depend on the validity of the "consensus block" it is contained within.

Reviewer: Commented elsewhere on this doc, but I feel strongly about us using full terms for the execution layer and consensus layer

Reviewer 2: I agree regarding EL/CL (though less strongly agree re: PoS/PoW, so maybe I'm a hypocrite).

Not setting a `fee recipient` will still allow your validator to behave as usual, but you will miss out on unburnt fee tips and any MEV you would have otherwise earned in blocks your validator proposes.

For more detailed information and a summary of links to client resources, stakers are encouraged to check out the [Merge Readiness Checklist](https://launchpad.ethereum.org/en/merge-readiness/) over on the Staking Launchpad to make sure you're fully prepared for The Merge.

Note for stakers using [SaaS](/staking/saas/) or [staking pools](/staking/pools/): There is nothing you need to do to prepare for The Merge. [More below on staying safe.](#users-holders)
</ExpandableCard>

<ExpandableCard
title="Non-validating node operators and infrastructure providers"
contentPreview="If you're operating a non-validating Ethereum node, the most significant change that comes with The Merge is the requirement to run clients for BOTH the execution layer AND the consensus layer."
id="node-operators">

> You probably are already running an execution layer client, such as Geth, Erigon, Besu or Nethermind. Up until The Merge, an execution layer client was enough to receive, properly validate, and propagate blocks being gossiped by the network. _After The Merge_, the validity of transactions contained within an execution payload will also depend on the validity of the "consensus block" it is contained within.

Reviewer: tense switching 

```suggestion
You probably are already running an execution layer client, such as Geth, Erigon, Besu or Nethermind. Until The Merge, an execution layer client will receive, properly validate, and propagate blocks being gossiped by the network. _After The Merge_, the validity of transactions contained within an execution payload will also depend on the validity of the "consensus block" containing the transaction.
```

Reviewer 3: not sure about this now. The validity of the transactions in the execution payload is independent of the beacon block wrapping them. The validity of the transactions is still determined by the execution client re-executing them locally and verifying that no rules are violated. Post-merge, transaction gossip, transaction execution, state-management, EVM etc are all still handled by the execution client.

"You are probably already running an execution client, such as Geth, Erigon, Besu or Nethermind. Until The Merge, the execution client alone is sufficient for interacting with the Ethereum blockchain. After The Merge, the execution client hands over responsibility for block building, block gossip and all consensus logic to a consensus client."

> This means that a full Ethereum node after The Merge requires both an execution layer client as well as a consensus layer client. These two clients work tightly together using a new Engine API to properly determine the latest state of the network. The Engine API requires authentication using a JWT secret, which is provided to both clients allowing secure communication.

Reviewer: unclear what 'this' refers to

```suggestion
As a result, a full Ethereum node after The Merge requires both an execution layer client and a consensus layer client. These two clients work together using a new Engine API to correctly determine the latest state of the network. The Engine API requires authentication using a JWT secret, which is provided to both clients allowing secure communication.
```

Reviewer 3: "...a full Ethereum node after The Merge **comprises** both an execution..."

do we have to use "execution *layer* client"? 

The engine API is not really about state management - this is still in the remit of the execution client. The engine-API is about enabling the consensus client to make requests to the execution client and for the execution client to return execution payloads and other information back up to the consensus client.

Just kill the sentence at "...using a new Engine API"?

Author: @minimalsm "As a result" of what? 😆

Key action items include:

> - Installing a consensus layer client in addition to an execution layer client

Reviewer: ```suggestion
- Install a consensus layer client in addition to an execution layer client
```

> - Authenticating both execution layer and consensus layer clients with a shared JWT secret so they can securely communicate with one another.

Reviewer: ```suggestion
- Authenticate both execution layer and consensus layer clients with a shared JWT secret so they can securely communicate.
```

Not completing the above items in time for The Merge will result in your node appearing to be "offline" until both layers are synced and authenticated.

> Node operators can also check out the [Merge Readiness Checklist](https://launchpad.ethereum.org/en/merge-readiness/) on the Staking Launchpad for more information, as much of the details apply to all node operators.

Reviewer: ```suggestion
Node operators can also check out the [Merge Readiness Checklist](https://launchpad.ethereum.org/en/merge-readiness/) on the Staking Launchpad for more information, as many of the details apply to all node operators.
```

For more information, check out this blog post by Tim Beiko on [How The Merge Impacts Ethereum’s Application Layer](https://blog.ethereum.org/2021/11/29/how-the-merge-impacts-app-layer/).
</ExpandableCard>

### Users and holders {#users-holders}

<strong>You do not need to anything to protect your funds entering The Merge.</strong><br/><br/>

<em>This bears repeating</em>: As a user or holder of ETH or any other digital asset on Ethereum, <strong>you do not need to do anything with your funds or wallet before The Merge.</strong>

Despite swapping out proof-of-work, the entire history of Ethereum since genesis remains intact and unaltered after the transition to proof-of-stake. Any funds being protected by your own wallet before The Merge will still be accessible in the same manner after The Merge. No action is required to "upgrade" on your part.

As we approach The Merge of Ethereum Mainnet, **you should be on high alert for scams trying to take advantage of users during this transition.** Do not send your ETH anywhere in attempt to "upgrade to ETH2." There is no "ETH2" token, and there is nothing more you need to do for your funds to remain safe.

> [More on Ethereum security](/security/#eth2-token-scam)

Reviewer: This whole section should be a fourth expandable card IMO, users/holders are just another group who need to be ready for the merge (even though the 'steps' to be ready are less)

Author: Had originally placed them in there, but @nloureiro and I discussed this, and came to this for a few reasons.

First it visually didn't look great since the page ended up mostly as cards. Mainly though, we want to reduce the burden for day-to-day users from seeing this information, which keeping it in a collapsed card would require them to click in.

Doing this for every category made the page way too long, and also those personas apply to a smaller population.

Reviewer 2: I agree w/ @minimalsm's initial point - making this one of the collapsable cards seems a better fit here.

To start, the initial text up top:

> For everyday users, holders, or non-node-operating stakers there is nothing you need to do, but a few things you should be aware of heading into The Merge. More on this below.

Seems less obvious/apparently because it doesn't stand out like the cards below. It's probably the most important text on the page. I feel like it's currently too underwhelming.

Then having a link that skips down to a section at the end (vs. just expanding a card) seems less intuitive.

I can understand the goal (reducing burden) but I don't think this current solution achieves that well. If not the first collapsable card, I would have entire "Users and holders" section above the others instead of breaking it up.

Author: @nloureiro ^

Author: I think moving the section up would be the best compromise. I disagree with the user section being collapsed.

## After The Merge {#after-the-merge}

This will signal the end of proof-of-work for Ethereum and start the era of a more sustainable, eco-friendly Ethereum. Learn more about [Ethereum energy consumption](/energy-consumption/).

This will also set the stage for further scalability upgrades not possible under proof-of-work, bringing Ethereum one step closer to achieving the full scale, security and sustainability outlined in its [Ethereum vision](/upgrades/vision/).

## Misconceptions about The Merge {#misconceptions}

<ExpandableCard

> title="Misconception: &quot;Running a node requires staking 32 ETH.&quot;"

Reviewer 2: Instead of repeating `Misconception:` a bunch of times, I might just rephrase these statements as questions, e.g.
- Does running a node require staking 32 ETH?
- Will The Merge will reduce gas fees?
- Can you withdraw staked ETH after The Merge?

Author: The goal here was to explicitly show these in terms of the things people are thinking, but are not correct. This approach was taken to try and catch peoples eyes, people who may think these things, but also make it absolutely clear these statements were false. @nloureiro If you have thoughts.

This is otherwise an FAQ again. We had spoke a bit about trying to make this more a "myth-busting" type section.
...Which does remind me, perhaps "myth" would be a shorter alternative here, at least in English.

> title="Misconception 1: &quot;Running a node requires staking 32 ETH.&quot;"

Reviewer: I don't see how numbering the misconceptions provides any value.

Author: @minimalsm Were you thinking just ditch the numbers? Or ditch the whole "Misconeption" lead-in as well?

Not necessarily against the former.

If the later, just know the reasoning for this in general was to try and minimize the possibility of someone skimming through and not realizing something was a _false_ statement. Open for suggestions... or a better shorter synonym.

Reviewer: I meant just ditching the numbers.

My first thought was to frame them as questions, but given the content, I'm not sure that makes sense... (having a list of questions where the answer is always false/no) 

I think the way you've got it is probably best.

> contentPreview="False. Anyone is free to sync their own self-verified copy of Ethereum, aka run a node. No ETH required—not before The Merge, not after The Merge, not ever.">

Reviewer: ```suggestion
contentPreview="False. Anyone is free to sync their own self-verified copy of Ethereum (i.e. run a node). No ETH is required. Not before The Merge, not after The Merge, not ever.">
```

There are two general types of Ethereum nodes: nodes that can propose blocks, and nodes that don't.

> Nodes that can propose blocks are only a _subset_ of the total nodes on the network. This category includes mining nodes under proof-of-work (PoW), and validator nodes under proof-of-stake (PoS). This category requires committing economic resources (such as GPU hash power in PoW, or staked ETH in PoS), in exchange for the ability to occasionally propose the next block and earn protocol rewards.

Reviewer: ```suggestion
Nodes that can propose blocks are only a _subset_ of the total nodes on the network. This category includes mining nodes under proof-of-work (PoW) and validator nodes under proof-of-stake (PoS). This category requires committing economic resources (such as GPU hash power in proof-of-work or staked ETH in proof-of-stake) in exchange for the ability to occasionally propose the next block and earn protocol rewards.
```

Running a "non block-producing" node is not only _possible_ for anyone under either consensus mechanism (PoW _or_ PoS), it is _strongly encouraged_ for all users to run their own node if possible. This is _immensely valuable_ for the network, and also gives added benefits to any individual running their own software, such as improved security, privacy and censorship resistance.

The ability for anyone to run their own node is _absolutely essential_ to maintaining the decentralization of the Ethereum network.

[More on running your own node](/run-a-node/)

</ExpandableCard>

<ExpandableCard
title="Misconception: &quot;The Merge will reduce gas fees.&quot;"
contentPreview="False. The Merge is a change of consensus mechanism, not an expansion of network capacity, and will not result in lower gas fees.">
Gas fees are a product of network demand relative to the capacity of the network. The Merge deprecates the use of proof-of-work, transitioning to proof-of-stake for consensus, but does not significantly change any parameters that directly influence network capacity or throughput.

> With a [rollup-centric roadmap](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698), efforts are being focused on scaling user activity at layer 2, while enabling layer 1 Mainnet as a secure decentralized settlement layer optimized for rollup data storage to help make rollup transactions exponentially cheaper. The transition to proof-of-stake is a critical precursor to realizing this.

Reviewer 2: Seems like a good opportunity to link to https://ethereum.org/en/layer-2/

Reviewer 2: I think also worth linking to "more on gas and fees" to explain if folks aren't familiar https://ethereum.org/en/developers/docs/gas/

Author: ```suggestion
With a [rollup-centric roadmap](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698), efforts are being focused on scaling user activity at [layer 2](/layer-2/), while enabling layer 1 Mainnet as a secure decentralized settlement layer optimized for rollup data storage to help make rollup transactions exponentially cheaper. The transition to proof-of-stake is a critical precursor to realizing this.
```

</ExpandableCard>

<ExpandableCard
title="Misconception: &quot;Transactions will be noticeably faster after The Merge.&quot;"

> contentPreview="False. Though some slight changes exist, transaction speed will mostly remain the same on layer 1.">

Reviewer: Should the context preview be clearer here? I think the crux of it is:

transaction speeds will improve slightly but not in a way that will be noticeable to end-users.

> The "speed" of a transaction can be measured a few ways, including time to be included in a block, and also time to "finalization." Both of these change slightly, but not in a way that will be felt significantly by users.

Reviewer: ```suggestion
A transaction's "speed" can be measured in a few ways, including time to be included in a block and time to finalization. Both of these changes slightly, but not in a way that users will notice.
```

> Historically with proof-of-work (PoW) block times target ~13.3 seconds. On the Beacon Chain, slots occur every 12 seconds exactly, each of which is an opportunity for a validator to publish a block. Most slots have blocks, but not necessarily all (ie. a validator is offline). In most cases, blocks will be produced ~10% more frequently under PoS than under PoW, but this is a fairly insignificant change in the scaling game, and unlikely to be noticed by users.

Reviewer: > each of which is an opportunity for a validator to publish a block.

is this necessary here?

> Historically with proof-of-work (PoW) block times target ~13.3 seconds. On the Beacon Chain, slots occur every 12 seconds exactly, each of which is an opportunity for a validator to publish a block. Most slots have blocks, but not necessarily all (ie. a validator is offline). In most cases, blocks will be produced ~10% more frequently under PoS than under PoW, but this is a fairly insignificant change in the scaling game, and unlikely to be noticed by users.

Reviewer: ```suggestion
Historically, on proof-of-work, the target was to have a new block every ~13.3 seconds. On the Beacon Chain, slots occur precisely every 12 seconds, each of which is an opportunity for a validator to publish a block. Most slots have blocks, but not necessarily all (i.e. a validator is offline). On proof-of-stake blocks will be produced ~10% more frequently than on proof-of-work. This is a fairly insignificant change and is unlikely to be noticed by users.
```

> PoS does introduce the concept of transaction "finality" which did not previously exist. Under PoW, the ability to reverse a block gets exponentially more difficult with every passing block that is mined on top of your transaction, but it never quite reaches zero. Under PoS, blocks are bundled into epochs (6.4 minute spans of time containing 32 chances for blocks) which are voted on by validators. After an epoch ends, it can be voted on as "justified," and then one more epoch later results in a "finalized" state. To undo these transactions would require obtaining and burning over 1/3 the total ETH staked.

Reviewer: Not strongly opposed to keeping this in, but this is overexplaining.

> PoS does introduce the concept of transaction "finality" which did not previously exist. Under PoW, the ability to reverse a block gets exponentially more difficult with every passing block that is mined on top of your transaction, but it never quite reaches zero. Under PoS, blocks are bundled into epochs (6.4 minute spans of time containing 32 chances for blocks) which are voted on by validators. After an epoch ends, it can be voted on as "justified," and then one more epoch later results in a "finalized" state. To undo these transactions would require obtaining and burning over 1/3 the total ETH staked.

Reviewer: ```suggestion
Proof-of-stake introduces the transaction finality concept that did not previously exist. On proof-of-work, the ability to reverse a block gets exponentially more difficult with every passing block mined on top of a transaction, but it never quite reaches zero. Under proof-of-stake, blocks are bundled into epochs (6.4 minute spans of time containing 32 chances for blocks) which validators vote on. When an epoch ends, validators vote on whether to consider the epoch 'justified'. If validators agree to justify the epoch, it gets finalized in the next epoch. Undoing finalized transactions is economically unviable as it would require obtaining and burning over one-third of the total staked ETH.
```

> Many dapps require a number of PoW block confirmations that take a period of time on par with how long PoS finality takes. Finality can offer additional security guarantees, but will not significantly speed up transactions.

Reviewer: Same as above. This is answering a different question than the one asked: "_Will transactions will be noticeably faster after The Merge?_"

```suggestion
Many dapps require a number of proof-of-work block confirmations that take a period of time on par with how long proof-of-stake finality takes. Finality can offer additional security guarantees, but will not significantly speed up transactions.
```

</ExpandableCard>

<ExpandableCard
title="Misconception: &quot;You can withdraw staked ETH once The Merge occurs.&quot;"

> contentPreview="False. Staking withdrawals are not yet enabled with The Merge. These will have to wait until the Shanghai upgrade to follow.">

Reviewer: > These will have to wait until the Shanghai upgrade to follow.

Sounds like we're putting people off

```suggestion
contentPreview="False. Staking withdrawals are not yet enabled with The Merge. The following Shanghai upgrade will enable staking withdrawals.">
```

> Staked ETH, staking rewards to date, and newly issued ETH immediately after The Merge will still be locked on the Beacon Chain without the ability to withdraw.

Reviewer: > newly issued ETH immediately after The Merge

This could be potentially confusing to users. They might assume that all 'new' ETH (e.g. ETH they buy from an exchange post-merge) is not accessible

> Withdrawal functionality is planned for the **Shanghai upgrade** to follow The Merge. This means that newly issued ETH, though accumulating on the Beacon Chain, will remain locked and illiquid for at least 6-12 months following The Merge.

Reviewer: Same as above. 

```suggestion
Withdrawals are planned for the Shanghai upgrade, the next major upgrade following The Merge. This means that newly issued ETH, though accumulating on the Beacon Chain, will remain locked and illiquid for at least 6-12 months following The Merge.
```

</ExpandableCard>

<ExpandableCard
title="Misconception: &quot;Validators will not receive any liquid ETH rewards til the Shanghai upgrade when withdrawals are enabled.&quot;"
contentPreview="False. Fee tips/MEV will be credited to a Mainnet account controlled by the validator, available immediately.">
This may seem counterintuitive to the above note that withdrawals are not enabled til the Shanghai upgrade, but validators WILL have immediate access to the fee rewards/MEV earned during block proposals.

> ETH is issued by the protocol as a reward to validators for contributing to consensus. This _newly issued_ ETH is accounted for on the Beacon Chain (consensus layer), where a validator has a unique address that holds its staked ETH and protocol rewards. This ETH is locked until Shanghai.

Reviewer: ```suggestion
The protocol issues ETH as a reward to validators for contributing to consensus. This Beacon Chain accounts for the newly issued ETH, where a validator has a unique address that holds its staked ETH and protocol rewards. This ETH is locked until Shanghai.
```

> ETH on the _execution layer_ (Mainnet as we know it) is accounted for separately from the consensus layer. When users execute transactions that involve Mainnet, ETH must be paid to cover the gas which includes a tip to the validator. This ETH is already on the execution layer, is NOT being newly issued by the protocol, and is available to the validator immediately (given a proper `fee recipient` address is provided to the client software).

Reviewer: ```suggestion
ETH on the execution layer (Ethereum Mainnet as we know it today) is accounted for separately from the consensus layer. When users execute transactions on Ethereum Mainnet, ETH must be paid to cover the gas, including a tip to the validator. This ETH is already on the execution layer, is NOT being newly issued by the protocol, and is available to the validator immediately (given a proper `fee recipient` address is provided to the client software).
```

Not a hill I'd die on, but I'd prefer to remove the final sentence entirely here. This should answer the misconception, not educate validators on setup best practices.

```suggestion
ETH on the execution layer (Ethereum Mainnet as we know it today) is accounted for separately from the consensus layer. When users execute transactions on Ethereum Mainnet, ETH must be paid to cover the gas, including a validator tip. This ETH is already on the execution layer, is NOT being newly issued by the protocol, and is available to the validator immediately.
```

</ExpandableCard>

<ExpandableCard
title="Misconception: &quot;When withdrawals are enabled, stakers will all exit at once.&quot;"

> contentPreview="False. Validator exits are rate limited for security reasons.">

Reviewer: Rate-limiting is developer terminology. We should try to speak in common language where possible... 

A more accessible way of phrasing this might be: 'The number of validators that can leave at once is capped for security reasons'

Given how buried this is I'm not concerned about going through this FAQ entirely and fixing it for now but something to consider if we talk about this anywhere else.

> After the Shanghai upgrade enabled withdrawals, all validators will be incentivized to withdraw their staking balance above 32 ETH, as these funds to not add to yield, and are otherwise locked. Depending on the APR (determined by total ETH staked), they may be incentivized to exit their validator(s) to reclaim their entire balance, or potentially to stake even more using their rewards to earn more yield.

Reviewer 2: ```suggestion
After the Shanghai upgrade enabled withdrawals, all validators will be incentivized to withdraw their staking balance above 32 ETH, as these funds do not add to yield, and are otherwise locked. Depending on the APR (determined by total ETH staked), they may be incentivized to exit their validator(s) to reclaim their entire balance, or potentially to stake even more using their rewards to earn more yield.
```

> After the Shanghai upgrade enabled withdrawals, all validators will be incentivized to withdraw their staking balance above 32 ETH, as these funds do not add to yield, and are otherwise locked. Depending on the APR (determined by total ETH staked), they may be incentivized to exit their validator(s) to reclaim their entire balance, or potentially to stake even more using their rewards to earn more yield.

Reviewer: ```suggestion
After the Shanghai upgrade enabled withdrawals, all validators will be incentivized to withdraw their staking balance above 32 ETH, as these funds do not add to yield and are otherwise locked. Depending on the APR (determined by total ETH staked), they may be incentivized to exit their validator(s) to reclaim their entire balance or potentially stake even more using their rewards to earn more yield.
```

> An important caveat here, full validator exits are rate limited by the protocol, so only 6 validators may exit per epoch (every 6.4 minutes, so 1350/day, or only ~43,200 ETH/day, out of >10 million ETH staked). This rate limit adjusts depending on total ETH staked, and acts as a bottleneck to prevent a mass exodus of funds, and also prevents a potential attacker from using their stake to commit a slashable offense and then exiting their entire staking balance in the same epoch before the slashing penalty can be enforced.

Reviewer: That's a mouthful :p

> An important caveat here, full validator exits are rate limited by the protocol, so only 6 validators may exit per epoch (every 6.4 minutes, so 1350/day, or only ~43,200 ETH/day, out of >10 million ETH staked). This rate limit adjusts depending on total ETH staked, and acts as a bottleneck to prevent a mass exodus of funds, and also prevents a potential attacker from using their stake to commit a slashable offense and then exiting their entire staking balance in the same epoch before the slashing penalty can be enforced.

Reviewer: ```suggestion
An important caveat here, full validator exits are rate limited by the protocol, so only six validators may exit per epoch (every 6.4 minutes, so 1350 per day, or only ~43,200 ETH per day out of over 10 million ETH staked). This rate limit adjusts depending on the total ETH staked and prevents a mass exodus of funds. Furthermore, it prevents a potential attacker from using their stake to commit a slashable offense and exiting their entire staking balance in the same epoch before the protocol can enforce the slashing penalty.
```

> The APR is intentionally dynamic, allowing a market of stakers to find a balance of how much they're willing to be paid to help secure the network. When withdrawals are enabled, if the rate is too low, then validators will exit, at a rate limited by the protocol. Gradually this will raise the APR for everyone who remains, slowly attracting new or returning stakers yet again.

Reviewer: ```suggestion
The APR is intentionally dynamic, allowing a market of stakers to balance how much they're willing to be paid to help secure the network. When withdrawals are enabled, if the rate is too low, then validators will exit at a rate limited by the protocol. Gradually this will raise the APR for everyone who remains, attracting new or returning stakers yet again.
```

</ExpandableCard>

<ExpandableCard
title="Misconception: &quot;Staking APR is expected to triple after The Merge.&quot;"
contentPreview="False. More up-to-date estimations predict closer to a 50% increase in APR post-merge, not a 200% increase.">

> The APR for stakers is expected to increase post-merge. To understand by how much, it is important to recognize where this increase in APR is coming from. This does not come from an increase in protocol ETH issuance (<a href="/upgrades/merge/issuance/">ETH issuance after The Merge</a> is in fact decreasing by ~90%), but is instead a reallocation of transaction fees that will start going to validators instead of miners.

Reviewer: ```suggestion
The APR for stakers is expected to increase post-merge. To understand by how much, it is important to recognize where this increase in APR is coming from. This does not come from an increase in protocol ETH issuance (<a href="/upgrades/merge/issuance/">ETH issuance after The Merge</a> is decreasing by ~90%), but is instead a reallocation of transaction fees that will start going to validators instead of miners.
```

This will be a new separate source of revenue for validators when they propose blocks. As you can imagine, the amount of fees a validator receives is proportional to network activity at the time of their proposed block. The more fees being paid by users, the more fees validators will receive.

> Looking at recent blockchain activity, approximately 10% of all gas fees being paid are currently going to miners in the form of a tip, while the rest is burnt. Outdated predictions estimated this percentage to be much higher, and was calculated when network usage was at all time highs. Extrapolating the 10% number to average recent network activity, it is estimated that the APR for staking will increase to ~7%, approximately 50% higher than the base issuance APR at time of writing.

Reviewer 2: > at time of writing

Should we say when (e.g. June 2022)?

Author: ```suggestion
Looking at recent blockchain activity, approximately 10% of all gas fees being paid are currently going to miners in the form of a tip, while the rest is burnt. Outdated predictions estimated this percentage to be much higher, and was calculated when network usage was at all time highs. Extrapolating the 10% number to average recent network activity, it is estimated that the APR for staking will increase to ~7%, approximately 50% higher than the base issuance APR (as of June 2022).
```

</ExpandableCard>

<ExpandableCard
title="Misconception: &quot;The Merge will result in downtime of the chain.&quot;"
contentPreview="False. The Merge upgrade is designed to transition to proof-of-stake with zero downtime.">
An immense amount of work has been put into making sure the transition to proof-of-stake does not disrupt the network or its users.

> The Merge is akin to changing an engine on a rocketship mid-flight, and is designed to be performed without any need to pause anything during the switch. The Merge will be triggered by what is known as a TTD, or [terminal total difficulty](/glossary/#terminal-total-difficult), which is a cumulative measure of the total mining power that has gone into building the chain. When the time comes, and this criteria is met, blocks will go from being produced using proof-of-work in one block, to being produced by proof-of-stake in the next.

Reviewer: ```suggestion
The Merge is like changing an engine on a rocketship mid-flight and is designed to be performed without needing to pause anything during the switch. The Merge will be triggered by **[terminal total difficulty (TTD)](/glossary/#terminal-total-difficult)**, which is a cumulative measure of the total mining power that has gone into building the chain. When the time comes, and this criterion is met, blocks will go from being built using proof-of-work in one block to being built by proof-of-stake in the next.
```

Ethereum does not have downtime.
</ExpandableCard>

## Relationship between upgrades {#relationship-between-upgrades}

The Ethereum upgrades are all somewhat interrelated. So let’s recap how The Merge relates to the other upgrades.

### The Merge and the Beacon Chain {#merge-and-beacon-chain}

> The Merge represents the formal adoption of the Beacon Chain as the new consensus layer to the current Mainnet execution layer. Once The Merge happens, validators will be assigned to secure Ethereum Mainnet and mining [proof-of-work](/developers/docs/consensus-mechanisms/pow/) will no longer be a valid means of block production.

Reviewer: ```suggestion
The Merge represents the formal adoption of the Beacon Chain as the new consensus layer to the current Mainnet execution layer. Once The Merge happens, validators will be assigned to secure Ethereum Mainnet, and mining on [proof-of-work](/developers/docs/consensus-mechanisms/pow/) will no longer be a valid means of block production.
```

> Blocks will instead be proposed by validating nodes that have ether staked for the right to participate in consensus. These upgrades set the stage for future scalability upgrades including data sharding.

Reviewer: ```suggestion
Blocks will instead be proposed by validating nodes that have ether staked for the right to participate in consensus. These upgrades set the stage for future scalability upgrades, including data sharding.
```

<ButtonLink to="/upgrades/beacon-chain/">
  The Beacon Chain
</ButtonLink>

### The Merge and the Shanghai upgrade {#merge-and-shanghai}

> In order to simplify and maximize focus on a successful transition to proof-of-stake, The Merge upgrade will not include certain anticipated features such as the ability to withdraw staked ETH. The Shanghai upgrade is planned to follow The Merge, in which the ability to withdraw is slotted as a top priority.

Reviewer 2: ```suggestion
In order to simplify and maximize focus on a successful transition to proof-of-stake, The Merge upgrade will not include certain anticipated features such as the ability to withdraw staked ETH. The Shanghai upgrade is planned to follow The Merge, which will enable the ability to stakers to withdraw.
```

> The Shanghai upgrade is planned to follow The Merge, in which the ability to withdraw is slotted as a top priority.

Not a big fan of the wording:
> in which the ability to withdraw is slotted as a top priority.

Can we rephrase this to something more simple?

Stay up-to-date with the [Shanghai upgrade planning issue on GitHub](https://github.com/ethereum/pm/issues/450), or the [EF Research and Development Blog](https://blog.ethereum.org/category/research-and-development/). For those curious, learn more about [What Happens After The Merge](https://youtu.be/7ggwLccuN5s?t=101), presented by Vitalik at the April 2021 ETHGlobal event.

### The Merge and data sharding {#merge-and-data-sharding}

> Originally, the plan was to work on shard chains before The Merge—to address scalability. However, with the boom of [layer 2 scaling solutions](/layer-2/), the priority has shifted to swapping proof-of-work to proof-of-stake via The Merge.

Reviewer 2: ```suggestion
Originally, the plan was to work on shard chains before The Merge in order to address scalability. However, with the boom of [layer 2 scaling solutions](/layer-2/), the priority has shifted to swapping proof-of-work to proof-of-stake via The Merge.
```

> Originally, the plan was to work on shard chains before The Merge in order to address scalability. However, with the boom of [layer 2 scaling solutions](/layer-2/), the priority has shifted to swapping proof-of-work to proof-of-stake via The Merge.

Reviewer: I feel like this phrasing creates a false narrative. The Merge was prioritised before we had any real viable scaling solutions.

```suggestion
Originally, the plan was to work on shard chains before The Merge to address scalability. However, with the boom of [layer 2 scaling solutions](/layer-2/), the priority has shifted to swapping proof-of-work to proof-of-stake via The Merge.
```

Plans for sharding are rapidly evolving, but given the rise and success of layer 2 technologies to scale transaction execution, sharding plans have shifted to finding the most optimal way to distribute the burden of storing compressed calldata from rollup contracts, allowing for exponential growth in network capacity. This would not be possible without first transitioning to proof-of-stake.

<ButtonLink to="/upgrades/shard-chains/">

>   Data sharding

Reviewer: Where can I read the discussion about using data sharding > just 'sharding'?

My gut take is that because we're still sharding Ethereum we should just say 'sharding'. Prefixing 'data' just adds more complexity for users.

</ButtonLink>

> ## Further reading {#further-reading}

Reviewer: 🙏
